syntax = "proto3";

message Inventory {
    int32 id = 1;
    int32 product_id = 2;
    int32 stock_level = 3;
    string last_updated = 4;
}

message InventoryUpdate {
    int32 product_id = 1;
    int32 new_stock = 2;
}